### ISSUE TITLE
Issue with citation in property

### ISSUE THREAD

**Issue by lucasvreis** [andras-simonyi/org-cite-csl-activate#9]
Org-roam can use citations inside the property drawer, for setting ROAM_REFS. But when inside a property value, the rendered citation by `org-cite-csl-activate` looks weird:

![Captura de tela de 2022-07-13 22-27-21](https://user-images.githubusercontent.com/4229115/178867768-b437cc51-9b78-498e-84ef-d4c6115adae1.png)

Presumably, this library uses the `org-element` API to figure out the citation boundaries, but in this case `org-element` returns the property drawer boundaries instead. I'm not sure if this is a problem in `org-element` API, but I noticed that vanilla org-mode also fontifies citations in property drawer values.

**Comment by andras-simonyi** [comment]
Thanks for reporting this, will try to look into it in the near future.

**Comment by andras-simonyi** [comment]
I've experimented with it a bit and on my system the problem only comes up with `org-cite-csl-activate-render-all`, otherwise the fontification doesn't try to render the citation. In the case of `org-cite-csl-activate-render-all`, the problem is basically caused by the fact that officially Org property drawers cannot contain citations, so when a cite is detected by regex matching then its reported parent is not a citation object (because there isn't one). The easy fix is just not touching these citations at all -- would it be an acceptable solution in your opinion?

**Comment by lucasvreis** [comment]
Yes, I agree it's probably better to only render citations that are identified by org-element

**Comment by andras-simonyi** [comment]
I've merged a PR (#10) which hopefully solved the problem.
